docs/guides/platform/marketplace/deploy-gitea-with-marketplace-apps/index.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: deploy-Gitea-with-marketplace-apps
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Deploy Gitea on Linode using Marketplace Apps.'
+og_description: 'Deploy Gitea on Linode using Marketplace Apps.'
+keywords: ['Gitea','version control','git']
+tags: ["linode platform","version control system","marketplace","cloud-manager"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2021-01-04
+modified: 2021-01-04
+modified_by:
+  name: Linode
+title: "Deploy Gitea with Marketplace Apps"
+contributor:
+  name: Linode
+external_resources:
+- '[Gitea Documentation](https://docs.gitea.io/'
+aliases: ['/guides/deploy-gitea-with-one-click-apps/']
+---
+## Gitea Marketplace App
+
+[Gitea](https://gitea.io/) is a community managed, painless, self-hosted Git service. Gitea is a complete solution for all aspects of your software development. At its core, Gitea serves as your centralized remote Git repository. Gitea also features built-in tools that represent every task in your development workflow, from planning to testing to releasing.
+
+Self-hosting your software development with the Gitea Marketplace App offers total control of your codebase. At the same time, its familiar interface eases collaboration for you and your team. Gitea is one of the most popular self-hosted Git repository platforms, allowing you to benefit from a robust set of integrated tools and an active community.
+
+### Software Included
```

Comment:
We typically include this info at the bottom of the guide, after the end of installation instructions.